packages/host/webpack.config.js

```diff
@@ -71,7 +71,7 @@ module.exports = env => {
         },
         {
           test: /\.(woff|woff2|ttf|eot)$/,
-          use: 'file-loader?name=fonts/[name].[ext]!static',
+          use: 'file-loader?name=fonts/[name].[ext]',
```

Comment:
oops! fonts were not working on the demo server, turns out that windows won't serve files with `!` in the name 🙄